services/shhext/service_test.go

```diff
@@ -92,7 +92,7 @@ func (s *ShhExtSuite) SetupTest() {
 			InstallationID:          "1",
 			DataDir:                 os.TempDir(),
```

Comment:
i think it needs to use https://golang.org/pkg/io/ioutil/#TempDir for isolation. os.TempDir() returns default temporary directory right?